### ISSUE TITLE
'NoneType' object has no attribute 'windowFlags'

### ISSUE THREAD

**Issue by emacsomancer** [parkouss/webmacs#75]
I've tried it both through Arch's AUR package, and following the manual install instructions, and both ways when I try to run (e.g. `python -m webmacs.main`), it fails to run, displaying an error: `'NoneType' object has no attribute 'windowFlags'`.

**Comment by parkouss** [comment]
I'm answering here using an arch (manual install) and I don't see this error. I suspect your graphic card may be an issue there - could you give me the full python backtrace of the error, and some information about your graphic card (using lspci for example) please?

**Comment by emacsomancer** [comment]
Ah. I have an nVidia card in that machine, and I think there's been a kernel update since I've rebooted. That may be the issue. I'll look into this more tomorrow.

**Comment by parkouss** [comment]
@emacsomancer any news?

**Comment by emacsomancer** [comment]
@parkouss Yes, I tested it just now and it works on my Arch machine. I suspect it was an Nvidia issue (which, sadly, probably means every time the nvidia driver updates, I'll *have* to reboot to use Webmacs). 

But it seems to be working very nicely right now. (I'm having trouble getting it to work on GuixSD, but I'll likely open a different ticket for that if I can't figure it out.)

  It's like using Conkeror again!  I used Conkeror for so many years, it was sad when it became more or less unviable. Thanks for making this!
